polls/views.py

```diff
@@ -91,8 +95,13 @@ def test_func(self):
 
 
 class AdminAPIView(APIView):
-    authentication_classes = [TokenAuthentication]
-    permission_classes = [permissions.IsAuthenticated, permissions.IsAdminUser]
-
     def get(self, request):
         return Response({"data": "hello"}, status.HTTP_200_OK)
+
+
+class AnnotationListAPI(mixins.ListModelMixin, generics.GenericAPIView):
+    queryset = Annotation.objects.all()
+    serializer_class = AnnotationSerializer
+
+    def get(self, request, *args, **kwargs):
```

Comment:
There is no need to override get method because by default return a list.

You can use ListAPIView for getting the list.